manifests/m/MicrosoftCorporation/MicrosoftReportBuilder/15.0.19914.0/MicrosoftCorporation.MicrosoftReportBuilder.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: MicrosoftCorporation.MicrosoftReportBuilder

```

```suggestion
PackageIdentifier: Microsoft.ReportBuilder
```
A folder already exists for Microsoft as a publisher, so the simpler Identifier is better